src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        Reader reader = new Reader();
+        Writer writer = new Writer();
+        writer.createCsvFile(toFileName);
+        File readFile = new File(fromFileName);
+        File fromFile = new File(toFileName);
```

Comment:
The variable 'fromFile' is misleadingly named, as it refers to the destination file 'toFileName'. Rename this variable to 'toFile' to better reflect its purpose.